shimmingtoolbox/unwrap/prelude.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+"""
+Wrapper to FSL Prelude (https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FUGUE/Guide#PRELUDE_.28phase_unwrapping.29)
+"""
+
+import numpy as np
+import os
+import nibabel as nib
+import subprocess
+
+
+def prelude(wrapped_phase, mag, affine, mask=np.array([-1]), path_2_unwrapped_phase='./unwrapped_phase.nii',
+            is_unwrapping_in_2d=True, is_saving_nii=False):
```

Comment:
i would not input `is_unwrapping_in_2d` but check input data dim: if 3d --> use 3d option.

Reply:
Defaulted to False, it is significantly slower when not using a mask however. Also added ability to input 2d.